### PR TITLE
add image to user's crawler page

### PR DESCRIPTION
**Pull request by mrfinch**
use category-colour instead of static colour in post layout's breadcrumbs

Related to https://meta.discourse.org/t/improving-discourse-static-html-archive/112497

Let me know if you have any suggestions on what we can add to this page. Any other information we can show to the unauthenticated user? An image was already used in meta information for crawler so I have added in UI too.

Screenshot from 2019-04-06 10-54-40

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1,4 +1,7 @@
-<h2><%= @user.username %></h2>
+<div class='user-crawler'>
+	<img src='<%= get_absolute_image_url(@user.small_avatar_url) %>' alt='Image of <%= @user.username %>' title='<%= @user.username %>' />
```
Comment (ZogStriP) [inline app/views/users/show.html.erb]: The alt should be localized

Comment (mrfinch) [reply app/views/users/show.html.erb]: localization and me :bowing_man:, thanks @ZogStriP, I have fixed it now

File: config/locales/server.en.yml [hunk 2]
```diff
@@ -2225,6 +2225,7 @@ en:
       fixed_primary_email: "Fixed primary email for staged user"
       same_ip_address: "Same IP address (%{ip_address}) as other users"
       inactive_user: "Inactive user"
+    image_alt: "Image of %{username}"
```
Comment (SamSaffron) [inline config/locales/server.en.yml]: we don't need to carry this localization. Just have the alt be the username

File: app/views/users/show.html.erb [hunk 3]
```diff
@@ -1,4 +1,7 @@
-<h2><%= @user.username %></h2>
+<div class='user-crawler'>
+	<img src='<%= get_absolute_image_url(@user.small_avatar_url) %>' alt='<%= t('user.image_alt', username: @user.username) %>' title='<%= @user.username %>' />
```
Comment (SamSaffron) [inline app/views/users/show.html.erb]: we don't want an absolute image url here, we want a CDN url

Comment (mrfinch) [reply app/views/users/show.html.erb]: do we have any helper function to get a CDN URL? @SamSaffron

Comment (mrfinch) [reply app/views/users/show.html.erb]: used `UrlHelper.local_cdn_url`, let me know if there is an issue with it.

File: app/assets/stylesheets/desktop/topic-post.scss [hunk 4]
```diff
@@ -900,9 +900,6 @@ span.highlighted {
 }
 
 #breadcrumbs {
-  .badge-category-bg {
-    background-color: $secondary-high;
-  }
```
Comment (SamSaffron) [inline app/assets/stylesheets/desktop/topic-post.scss]: why is this being removed @awesomerobot  you ok with this?

File: app/views/topics/show.html.erb [hunk 5]
```diff
@@ -11,7 +11,7 @@
         itemref="breadcrumb-<%=(i+1)%>"
       <%-end%>>
       <a href="<%=c[:url]%>" itemprop="url" class='badge-wrapper bullet'>
-        <span class="badge-category-bg"></span>
+        <span class="badge-category-bg" style='background-color: #<%= c[:color] %>'></span>
```
Comment (SamSaffron) [inline app/views/topics/show.html.erb]: how is this going to work? where is the foreground? 

My call is not to even style this cause its just too annoying.

**Comment by SamSaffron** [comment]
I am super sorry but I am closing this, there are two changes being proposed here 

1. Adding image to user page in crawler view
2. Fiddle with styling of categories in crawler view 

We need 2 distinct PRs here.

**Pull request closed without merging**